controllers/openstack_ansibleee_controller.go

```diff
@@ -118,6 +118,8 @@ func (r *OpenStackAnsibleEEReconciler) Reconcile(ctx context.Context, req ctrl.R
 			instance.Status.Conditions.Set(instance.Status.Conditions.Mirror(condition.ReadyCondition))
 		}
 
+		instance.Status.ObservedGeneration = instance.Generation
```

Comment:
you can set it with the annotation:
https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/main/api/v1beta1/openstack_ansibleee_types.go#L143